### PR TITLE
Ability to edit description

### PR DESCRIPTION
**Pull request by mattvv**
wip for editing markdown.

putting up here as draft because i'm gonna finish tomorrow.

I think everything is in place, but beacuse this markdown doesn't support a live change of value, I need to figure out how to make a new instance of the markdown editor after change. Was hoping key would do it but it appears not.

### CODE REVIEW

File: components/MarkdownEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const globalTheme = useSelector(getTheme);
+  const modifiedDark = { ...dark, background: 'transparent' };
+  const markdownTheme = globalTheme === 'dark' ? modifiedDark : light;
+
+  //todo: figure out why we can't refresh
+
+  return (
+    <Editor
+      key={defaultValue}
+      defaultValue={defaultValue}
```
Comment (mattvv) [inline components/MarkdownEditor.tsx:30]: After saving, the edit event is triggered but this default value is getting cached, todo: figure out how to go around it

Comment (mattvv) [reply components/MarkdownEditor.tsx:28]: this seems fixed now

File: components/StoryModal.tsx [hunk 2]
```diff
@@ -29,13 +33,32 @@ const StoryModal = (): JSX.Element => {
   const dispatch = useDispatch();
   const isOpen = useSelector(isStorySelected);
   const story = useSelector(getSelectedStory);
+  const [description, setDescription] = useState<string>(story?.description);
+
+  console.log('render with description', description);
```
Comment (diegofigs) [inline components/StoryModal.tsx:38]: Log here

**Review by diegofigs — APPROVED** [review]
One log statement; LGTM

File: components/StoryModal.tsx [hunk 3]
```diff
@@ -29,13 +33,32 @@ const StoryModal = (): JSX.Element => {
...
+
+  const [_, saveDescription] = usePivotal(async ({ apiKey, projectId }) => {
+    const newStory = await PivotalHandler.updateStory({
+      apiKey,
+      projectId,
+      storyId: story.id,
+      payload: { description },
+    });
+    console.log('saved story', newStory);
```
Comment (jansen0825) [inline components/StoryModal.tsx:47]: 👀

**Review by jansen0825 — APPROVED** [review]
Nice!

**Pull request merged**